apps/condo/domains/organization/schema/OrganizationEmployeeRole.test.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Generated by `createschema organization.OrganizationEmployeeRole 'organization:Relationship:Organization:CASCADE; name:Text; statusTransitions:Json; canManageOrganization:Checkbox; canManageEmployees:Checkbox; canManageRoles:Checkbox; canManageIntegrations:Checkbox; canManageProperties:Checkbox; canManageTickets:Checkbox;' --force`
+ */
+
+const { createTestOrganizationEmployee } = require('../utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+const { createTestOrganization } = require('../utils/testSchema')
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE } = require('@core/keystone/test.utils')
+
+const { OrganizationEmployeeRole, createTestOrganizationEmployeeRole, updateTestOrganizationEmployeeRole } = require('@condo/domains/organization/utils/testSchema')
+
+describe('OrganizationEmployeeRole', () => {
+    describe('user: create OrganizationEmployeeRole', () => {
+
+        it('can with granted "canManageRoles" permission', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [organization] = await createTestOrganization(admin)
+            const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                canManageRoles: true,
+            })
+            const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+            await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+            const [obj, attrs] = await createTestOrganizationEmployeeRole(managerUserClient, organization)
+
+            expect(obj.id).toMatch(UUID_RE)
+            expect(obj.dv).toEqual(1)
+            expect(obj.sender).toEqual(attrs.sender)
+            expect(obj.v).toEqual(1)
+            expect(obj.createdBy).toEqual(expect.objectContaining({ id: managerUserClient.user.id }))
+            expect(obj.updatedBy).toEqual(expect.objectContaining({ id: managerUserClient.user.id }))
+            expect(obj.createdAt).toMatch(DATETIME_RE)
+            expect(obj.updatedAt).toMatch(DATETIME_RE)
```

Comment:
check another fields, user, email, phone ...

Reply:
Added in `79c1830e`